libyul/AsmAnalysis.cpp

```diff
@@ -352,77 +361,86 @@ size_t AsmAnalyzer::operator()(FunctionCall const& _funCall)
 	std::optional<size_t> numReturns;
 	std::vector<std::optional<LiteralKind>> const* literalArguments = nullptr;
 
-	if (BuiltinFunction const* builtin = resolveBuiltinFunction(_funCall.functionName, m_dialect))
```

Comment:
this returns non-null IFF the `_funCall.functionName` variant holds a `BuiltinName`